paper/src/main/java/com/denizenscript/denizen/paper/events/EntityKnocksbackEntityScriptEvent.java

```diff
@@ -33,6 +36,7 @@ public class EntityKnocksbackEntityScriptEvent extends BukkitScriptEvent impleme
     // <context.entity> returns the EntityTag that was knocked back.
     // <context.damager> returns the EntityTag of the one who knocked.
     // <context.acceleration> returns the knockback applied as a vector.
+    // <context.cause> returns the cause of the knockback (only on 1.20+). Causes list: <@link url https://jd.papermc.io/paper/1.21.1/io/papermc/paper/event/entity/EntityKnockbackEvent.Cause.html>

```

Comment:
`MC 1.20+`, with the `MC` prefix. The version 1.20 of *what*? Version 1.20 of Denizen? Version 1.20 of Paper? Version 1.20 of Windows?